testdata/time.go

```diff
@@ -11,3 +11,27 @@ import (
 var rpcTimeoutMsec = flag.Duration("rpc_timeout", 100*time.Millisecond, "some flag") // MATCH /Msec.*\*time.Duration/
 
 var timeoutSecs = 5 * time.Second // MATCH /Secs.*time.Duration/
+
+var delay = flag.Duration("delay", 5, "wait impatiently") // MATCH /time\.Duration without unit: 5/
+
+func f(...time.Duration) {}
+
+var (
+	_ = time.Duration(15)
```

Comment:
`time.Duration(30)` now gets flagged. There are some other cases that get missed; there is a discussion in the comment in the newest commit.